woocommerce/woocommerce-android
# Feature/9649 code clean up store optimization

Closes: #9649 


### Description

Removes features flags related to store creation profiler steps. Additionally, it removes code that is not used anymore. 

### Testing instructions

1. Trigger store creation flow
2. Create a new store and check all the flow works as expected. 
3. Check that the following logs are shown: 
```
Profiler Answers uploaded successfully

🔵 Tracked: site_creation_profiler_data, Properties: {"industry_slug":"electronics_and_computers","user_commerce_journey":"im_just_starting_my_business","ecommerce_platforms":"","country_code":"AD","is_debug":true}
```
